pyquil/tests/test_api.py

```diff
@@ -234,7 +247,7 @@ def test_prepare_register_list():
 # ---------------------
 
 
-def test_get_qc_returns_remote_qvm_compiler():
+def test_get_qc_returns_remote_qvm_compiler(qvm: QVMConnection, compiler: QVMCompiler):
     with patch.dict('os.environ', {"COMPILER_URL": "tcp://192.168.0.0:5550"}):
```

Comment:
This isn't something you touched, but this line puzzles me. Is this connecting to an actual mock? Is it connecting to nothing and just checking that `get_qc` behaves as expected? Does the constructor still do a version check? Is it going to expect something to be there? If it's actually connecting to something, what does `192.168.0.0` specifically signify? That looks like a weird broadcast-type address to me.

Reply:
I think we need to do a larger UT audit for pyQuil